models.py

```diff
@@ -19,12 +15,12 @@ def __init__(self):
         self.losses = 0
         self.rating = 1500
 
+    @property
```

Comment:
what does this do

Reply:
Look it up

A method `m` annotated with `@property` can be accessed via `.m` instead of `.m()`.

Also, you can make a setter, but I don't use that.